### PR TITLE
Fix misc bugs

### PR DESCRIPTION
**Pull request by ethanbao**
1) duplicate remote logging
2) syntax error in CONTRIBUTING.rst
3) mark CLOUD_LOGGING_CLIENT global to avoid use of local variable before assignment error.
4) install less

### CODE REVIEW

File: artman/conductors/cloudtasks_conductor.py [hunk 1]
```diff
@@ -120,14 +120,14 @@ def _delete_task(task_client, task):
...
     """Setup logger with one-time logging FileHandler."""
-    logger = logging.getLogger('artman')
     log_file_handler = logging.FileHandler(log_path)
     logger.addHandler(log_file_handler)
     return log_file_handler
 
 
 def _write_to_cloud_logging(log_id, log_file_path):
     """Write log file content to cloud logging"""
+    global CLOUD_LOGGING_CLIENT
```
Comment (geigerj) [inline artman/conductors/cloudtasks_conductor.py]: Why global? Why is conductor not an object that has a logging client as an instance variable?

File: artman/conductors/cloudtasks_conductor.py [hunk 2]
```diff
@@ -120,14 +120,16 @@ def _delete_task(task_client, task):
...
-    logger = logging.getLogger('artman')
     log_file_handler = logging.FileHandler(log_path)
     logger.addHandler(log_file_handler)
     return log_file_handler
 
 
 def _write_to_cloud_logging(log_id, log_file_path):
     """Write log file content to cloud logging"""
+    # TODO(ethanbao): Turn conductor into a python object so that the logging
+    # client can be instance variable not global variable.
```
Comment (ethanbao) [inline artman/conductors/cloudtasks_conductor.py]: @geigerj Added a TODO.

Comment (geigerj) [inline artman/conductors/cloudtasks_conductor.py]: Could you file an issue instead?

Comment (ethanbao) [inline artman/conductors/cloudtasks_conductor.py]: https://github.com/googleapis/artman/issues/226

**Pull request merged**